### DISCUSSION TITLE
GTX 7 support?

### DISCUSSION THREAD

**Discussion by kvirulent** [terminatorul/NvStrapsReBar#72] [General]
Just wondering if GTX 700 series GPUs will be supported because I'm using a GTX 750 Ti.

**Comment by terminatorul** [comment]
Another user reported the same problem with a Maxwell 2 card (you must have a Maxwell 1 card) as with Pascal: https://github.com/terminatorul/NvStrapsReBar/discussions/28

NvStrapsReBar can enable a large BAR on the PCI bus for the GPU, but when Windows boots, the NVIDIA driver for Windows then reports a problem code or triggers a blue screen.

But since you have Maxwell 1, it is worth trying to enable ReBAR using NvStrapsReBar, and report your results here. Remember to set back the current year in UEFI Setup to disable NvStrapsReBar, in case you have no iGPU and can not boot Windows

**Reply by kvirulent** [reply]
Ah, ignore me. I haven't tried to use NvStrapsReBar yet, I thought it would be incompatible with my 750 Ti because the readme says 1600/2000 series GPUs, but looking at #1 a GT 640 is on there so maybe I was wrong. Woops!
